packages/clay-core/src/vertical-nav/VerticalNav.tsx

```diff
@@ -34,6 +34,16 @@ const Trigger = ({
 	</Button>
 );
 
+const Menubar = ({collection, value}) => {
```

Comment:
@matuzalemsteles `yarn lint:typescript` is complaining here. 

```
packages/clay-core/src/vertical-nav/VerticalNav.tsx:37:19 - error TS7031: Binding element 'collection' implicitly has an 'any' type.

37 const Menubar = ({collection, value}) => {
                     ~~~~~~~~~~

packages/clay-core/src/vertical-nav/VerticalNav.tsx:37:31 - error TS7031: Binding element 'value' implicitly has an 'any' type.

37 const Menubar = ({collection, value}) => {
```
I don't know how to fix it.

Reply:
I'll fix this, push a commit to deal with this and the const.